R/rename_ard_columns.R

```diff
@@ -0,0 +1,130 @@
+#' Rename ARD Variables
+#'
+#' Rename the grouping and variable columns to their original column names.
+#'
+#' @param x (`data.frame`)\cr
+#'   an ARD data frame of class 'card'
+#' @param columns ([`tidy-select`][dplyr::dplyr_tidy_select])\cr
+#'   columns to rename, e.g. selecting columns `c('group1', 'group2', 'variable')`
+#'   will rename `'group1_level'` to the name of the variable found in `'group1'`.
+#'   When, for example, the `'group1_level'` does not exist, the values of the
```

Comment:
this implies the `fill` applies to all variables, not just variable. I think this is how it worked previously, but not anymore? is that right?

Reply:
Hmmm, I think my wording of the argument description isn't super clear, but I am not entirely sure what you mean here either 😆 

I think in reality, the only two combinations of arguments that will every be passed here are `c(all_ard_groups("names"), all_ard_variables("names"))` (called by Roche users), and `all_ard_groups("names")` called when you don't want to touch the primary summary variables like you would need for tfmrt.

Here, we are just selecting the columns that will be renamed, e.g. selecting `group1` would combine group1 and group1_level into a single column.

Reply:
Makes sense!!! I was focused on this part " When, for example, the `'group1_level'` does not exist, the values of the new column are filled with the values in the `fill` argument." The rename function *used* to fill the group levels but doesnt anymore. so does this language need updating?